client/src/components/arabicListWords/index.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react'
+import axios from 'axios'
+import './style.css'
+
+class arabicListWords extends Component {
+  state = {
+    arabicList: []
+  }
+
+  componentDidMount() {
+    axios.get(`api/arabic-words`).then(({ data }) => {
+      this.setState({ arabicList: data })
+    })
+  }
+
+  render() {
+    return (
+      <div>
+        <h1> List Of Arabic Words </h1>
+        <React.Fragment>
+          {this.state.arabicList.map(element => (
+            <ul>
+              <li className="list">
+                <div>
```

Comment:
I'm confused about this structure, why are you making a ul ordered list and then divs? I think you just need to create divs here.

Reply:
Ok, but I want to think how I need to display the list so when I make a CSS I solve this.